scripts/cloudize-workflow.py

```diff
@@ -293,6 +293,12 @@ def cloudize(bucket, wf_path, inputs_path, output_path, dryrun=False):
     and its workflow's CWL definition."""
     workflow = make_workflow(wf_path, inputs_path)
     file_inputs = workflow.find_file_inputs()
+
+    if (not bool(file_inputs)):
+        logging.error(f"file_inputs is empty {file_inputs}. "
+        "Check that your LSF_DOCKER_VOLUMES is set up correctly within your .bashrc")
```

Comment:
I think we should make this comment a bit more generic.  I think this LSF_DOCKER_VOLUMES thing is very particular to our cluster.  There are various reasons why a file might not be accessible.